index.src.html

```diff
@@ -2665,6 +2649,10 @@
 but audio content using 24-bit samples is common, so 32-bits integers are
 commonly used to hold 24-bit content.
 
+{{AudioDecoder}}s decoding 24-bit audio chunks MUST convert their output to
+{{s32}} or {{s32-planar}}, by left-shifting the decoded samples by `8` before
```

Comment:
Add a note that explains what happens when the audio clips.

Mentions that we're talking about a bit shift on each sample: we're specifically not shifting the decoded samples by 8, we're bit-shifting each sample by 8.

Add a note that explain why this is OK, and the trade-off: any audio that clips cannot be decoded losslessly. Mention that reading back to f32 doesn't clip, and that authors that want this to work want to read-back to float. Mention that implementations SHOULD take care to not clip internally (we can discuss the normative requirement, of course).

Reply:
Taking over for @tguilbert-google since he's OOO this week.

The current text indicates that _AudioDecoders_ should always convert 24-bit audio to 32-bit audio via bit-shifting, so even if copyTo() is called later with float32, the contents will already have been clipped.

I've updated the text to be more inline with your thoughts. PTAL.